src/components/Form/MucSelect.vue

```diff
@@ -173,32 +189,49 @@ const clicked = (clickedValue: string) => {
  * Update the modelValue with the given. Performs conversion to string if necessary.
  * @param newValue the new value
  */
-const updateMVSingle = (newValue: string) => {
-  if (typeof selectedValues.value === "object")
+const updateMVSingle = (newValue: string | ItemAsObject) => {
+  if (Array.isArray(selectedValues.value))
     selectedValues.value = selectedValues.value.join(" ");
 
-  selectedValues.value = selectedValues.value === newValue ? "" : newValue;
+  selectedValues.value =
+    JSON.stringify(selectedValues.value) === JSON.stringify(newValue)
+      ? ""
+      : newValue;
 };
 
 /**
  * Update the modelValue with the given. Performs conversion to array if necessary.
  * @param newValue the new value
  */
-const updateMVMultiple = (newValue: string) => {
-  if (typeof selectedValues.value === "string")
+const updateMVMultiple = (newValue: string | ItemAsObject) => {
+  if (!Array.isArray(selectedValues.value))
     selectedValues.value = [selectedValues.value];
 
-  selectedValues.value = selectedValues.value.includes(newValue)
-    ? selectedValues.value.filter((val: string) => val !== newValue)
-    : [...selectedValues.value, newValue];
+  if (Array.isArray(selectedValues.value))
```

Comment:
Unfortunately I have to make this query, otherwise I get an error.
`Vue: Type any[] | ItemAsObject must have a [Symbol.iterator]() method that returns an iterator.`
Do you have a solution for this?